### PR TITLE
Added status.premid.app & astr.io

### LINKED ISSUE #1117: Rich Presence for the broswer game Astr.io - https://astr.io/
**Prerequisites and essential questions** 
* 

- [x] Is it a popular site?

* [x] Is the website older than 2 months? 
* [ ] Is the site locked to a specific country/region?
* [ ] Is the site a paid service? (e.g. Netflix, Hulu)
* [ ] Does the website feature NSFW content? (e.g. porn, etc...)
* [ ] Are you a donator/patron?
* [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** ⦂灸 Azrael#0031

**What's the name of the service?** 
https://astr.io/ - Astr.io
**What should the Presence display?**  It would be awesome for it show the server we are currently playing in and the region that server we are on for example: NA Domination I Mass: (current mass), and adding our current name we are using in-game to the presence 

**If possible, please provide a logo for the service (512x512 minimum)** 
cover-256 1

### PR DESCRIPTION
**Pull request by callumok2004**
image
image
image
image
closes #1117

### CODE REVIEW

**Comment by callumok2004** [comment]
image
image

**Comment by callumok2004** [comment]
#1117

**Comment by Azrael206** [comment]
Woah! This looks freaking sick, nice work man!

File: Astr.io/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "author": {
+    "name": "Callum",
+    "id": "506899274748133376"
+  },
+  "service": "Astr.io",
+  "description": {
+    "en": "A free to play web browser based game where users play as a planet and destroy others!",
+    "ga_IE": "Cluiche brabhsálaí gréasáin atá saor in aisce a sheinntear mar a imríonn úsáideoirí mar phláinéid agus a scriosann daoine eile!"
+  },
+  "url": ["astr.io"],
```
Comment (Bas950) [inline Astr.io/dist/metadata.json]: Shouldnt be in an array

File: Astr.io/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/jgsd4vO.png",
+  "thumbnail": "https://i.imgur.com/Sqk5GVy.png",
+  "color": "#3273dc",
+  "tags": [
+    "io-game",
+    "games"
+  ],
+  "category": "games",
+  "button": false
```
Comment (Bas950) [inline Astr.io/dist/metadata.json]: remove button: false
you are not allowed to add that

**Pull request merged**